apps/nextjs-website/package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "scripts": {
     "download-docs": "./scripts/fetch-docs.sh docs/from-gitbook",
+    "update-docs": "git submodule init && git submodule update && cd docs && git checkout docs/from-gitbook && git pull",
```

Comment:
You should move this commands into a shell script and use the temporary folder to `move` only `docs` subfolder in `apps/nextjs-website/docs/`. Right now with this script you create the nested subfolder `docs/docs` and the parser cannot access the manuals correctly.